app/views/govuk_publishing_components/components/_layout_header.html+govwales.erb

```diff
@@ -12,10 +12,10 @@
 
       <div class="header__components__language" id="wg_language">
         <%=
-          if locale == :en
```

Comment:
this looks ok, just wondering if there's any intentional reason/benefit for swapping the order around from if en to if cy ?

Reply:
This swap around allows us to remove the `elsif` (it's preferable to have a catch all `else` clause that an `elsif`)

Otherwise we'd ideally need a final `else` such as 

```ruby
if locale == :en
  # English
elsif locale == :cy 
  # Welsh 
else
  # A default, in case locale resolves to something else
end
```

Reply:
I get it now, thanks